### PR TITLE
:scroll: Update changelog. [skip ci]

### PR DESCRIPTION
**Pull request by kaikai1024**
### Description of the Change

Ready for the changelog of new version.

### CODE REVIEW

**Comment by kaikai1024** [comment]
@leeyr338 @ouwenkg 
Please review it and check the description.

**Comment by kaikai1024** [comment]
@KaoImin

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,51 @@ All notable changes to this project will be documented in this file. And this pr
...
+- [Optimization] Integrate vm-test. [@WPF]
+
+### Auth
+
+- [Fix] Auth crashes when it is not ready. [@Yaorong]
+
+### Network
+
+- [Fix] Refuse duplicate nodes. [@Yaorong]
+- [Fix] Node lostes. [@Yaorong]
```
[the remark this answers is not in the document]
Comment (kaikai1024) [reply CHANGELOG.md]: ok

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,51 @@ All notable changes to this project will be documented in this file. And this pr
...
+
+- [Optimization] Integrate vm-test. [@WPF]
+
+### Auth
+
+- [Fix] Auth crashes when it is not ready. [@Yaorong]
+
+### Network
+
+- [Fix] Refuse duplicate nodes. [@Yaorong]
```
[the remark this answers is not in the document]
Comment (kaikai1024) [reply CHANGELOG.md]: ok

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,51 @@ All notable changes to this project will be documented in this file. And this pr
...
+- [Feature] Add `peersInfoAdd ` interface. [@Yaorong]
+
+### Scripts
+
+- [Optimization] Installation && Exectution Optimization. [@clearloop]
+- [Fix] Redirect the stdout and stderr for daemon processes in docker. [@yangby]
+- [Optimization] Patch to absolute paths' in starting scripts. [@clearloop]
+- [Fix] Fix the `restart` command. [@kaikai]
+
+### Doc
```
Comment (leeyr338) [inline CHANGELOG.md]: The doc described here has nothing to do with citahub-docs, right?

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,51 @@ All notable changes to this project will be documented in this file. And this pr
...
+
+### Consensus
+
+- [Optimization] Set the default ntp service to false. [@kaikai]
+- [Fix] Not generate block. [@jerry-yu]
+
+### RPC
+
+- [Feature] Add `getVersion` interface. [@luqz]
+- [Feature] Add `peersInfoAdd ` interface. [@Yaorong]
```
[the remark this answers is not in the document]
Comment (kaikai1024) [reply CHANGELOG.md]: my fault: typo

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,49 @@ All notable changes to this project will be documented in this file. And this pr
 
 ## [Unreleased]
 
+### Framework
+
+- [Optimization] Update default rust toolchain. [@yangby]
+- [Optimization] Update cita-sdk-js version. [@kaikai]
+- [Fix] Reorganize toml path. [@WPF]
+- [Feature] Config the log module. [@kaiyu]
```
Comment (leeyr338) [inline CHANGELOG.md]: Is better change to `Make CITA's log output mode can be configured as stdout or log-file` ?

File: CHANGELOG.md (hunk 3 above)
Comment (kaikai1024) [reply CHANGELOG.md]: actually, it includes the citahub-docs.
it should include all the part of CITA.

something new including the missed part, or new structure

I will add more info later

should not be added:

* the new feature description, becasue doc should be the part it

File: CHANGELOG.md (hunk 5 above)
Comment (kaikai1024) [reply CHANGELOG.md]: nice. I will take part of it:

> Log output mode can be configured as stdout or log-file

**Comment by kaikai1024** [comment]
@KaoImin 先这样，之后可以随时补充

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,58 @@ All notable changes to this project will be documented in this file. And this pr
...
+### Auth
+
+- [Fix] Auth crashes when it is not ready. [@leeyr338]
+
+### Network
+
+- [Fix] High CPU usage. [@leeyr338]
+- [Fix] Refuse connect when reach max connections. [@leeyr338]
+- [Optimization] Add discovery test of network. [@leeyr338]
+- [Optimization] Use new version `p2p` API. [@jerry-yu]
```
Comment (leeyr338) [inline CHANGELOG.md]: Use new version `p2p` can fix this issue, It is better clearly written here.
https://github.com/cryptape/cita/issues/472

Comment (kaikai1024) [reply CHANGELOG.md]: ok

File: CHANGELOG.md [hunk 7]
```diff
@@ -4,6 +4,59 @@ All notable changes to this project will be documented in this file. And this pr
...
+
+- [Optimization] Installation && Exectution Optimization: new usage of cita script. [@clearloop]
+- [Fix] Redirect the stdout and stderr for daemon processes in docker. [@yangby-cryptape]
+- [Optimization] Patch to absolute paths' in starting scripts. [@clearloop]
+- [Optimization] Format the `env.sh` using `ShellCheck`. [@clearloop]
+
+### Doc
+
+- [Doc] Add style guide of codes. [@kaikai1024]
+- [Doc] Add all all contributors. [@kaikai1024]
```
Comment (ouwenkg) [inline CHANGELOG.md]: Repeated all ?

Comment (kaikai1024) [reply CHANGELOG.md]: done

**Pull request merged**